pkgs/by-name/co/commafeed/common.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, fetchFromGitHub }:
+let
+  version = "4.3.3";
+in
+{
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "Athou";
+    repo = "commafeed";
+    rev = version;
+    hash = "sha256-y0gTmtlDg7sdunG1ne/3WkFx2KQkTGRlfYpXBHFFh2o=";
+  };
+
+  npmDepsHash = "sha256-fye7MPWXUeFCMgcnesspd1giGG/ZldiOv00fjtXZSb4=";
+
+  mvnHash = "sha256-YnEDJf4GeyiXxOh8tZZTZdLOJrisG6lmShXU97ueGNE=";
```

Comment:
It would probably be easier and simpler if the hashes where at the specific package and the src is just inherited from the main package into the frontent.

Reply:
I moved everything to ``package.nix``. After your comment I came to the conclusion that using separate files in this package doesn't give anything special. 1 file is simpler.